src/main/java/seedu/address/model/student/Assessment.java

```diff
@@ -40,14 +43,18 @@ public static boolean isValidAssessment(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
-    public String getValue() {
-        return value;
+    public String getName() {
+        return name;
     }
 
     public Map<ID, Score> getScores() {
         return scores;
     }
 
+    public AssessmentStatistics getStatistics() {
+        return statistics;
+    }
```

Comment:
my slight concern with this is someone may use getStatistics() without doing setStatistics() first, then they end up using outdated statistics. 

```suggestion
    public AssessmentStatistics getStatistics() {
        setStatistics();
        return statistics;
    }
```

Reply:
Oh right that makes sense 👍🏻